security-command-center/snippets/src/main/java/vtwo/bigquery/CreateBigQueryExport.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo.bigquery;
+
+// [START securitycenter_create_bigquery_export_v2]
```

Comment:
This tag should enclose only the function `public static void createBigQueryExport(`, no?

Reply:
it makes sense if we follow the same example we used in GetIamPolicies:
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/security-command-center/snippets/src/main/java/vtwo/iam/GetIamPolicies.java

but I'm worried about the change, as the same PR is already approved in the same syntax.
https://github.com/GoogleCloudPlatform/java-docs-samples/pull/9291

Reply:
ok, no problem